src/App.tsx

```diff
@@ -1,12 +1,16 @@
 import { Route, Routes } from 'react-router-dom'
 
 import { HomePage } from './pages/HomePage'
+import { RegisterPage } from './pages/RegisterPage'
+import { SignInPage } from './pages/SignInPage'
 
 export function App(): JSX.Element {
   return (
-    <div className="App">
+    <div className="h-screen">
       <Routes>
-        <Route path="/" element={<HomePage />}></Route>
+        <Route path="/" element={<HomePage></HomePage>}></Route>
+        <Route path="/signin" element={<SignInPage></SignInPage>}></Route>
```

Comment:
```suggestion
        <Route path="/signin" element={<SignInPage />}></Route>
```
лучше коротко писать, если не кидаешь children.